### PR TITLE
Update go path to handler

### PR DESCRIPTION
**Pull request by andy-mcgrath**
Update to allow for `go get github.com/leesutcliffe/myip` and `go install github.com/leesutcliffe/myip` to work

### CODE REVIEW

**Review by leesutcliffe — CHANGES_REQUESTED** [review]
Handlers package is intended to be local to the project. 
Adding `go.mod` in the project root should also allow `go build` to run without file name

``` go
//go.mod
module myip
go 1.15
```

File: main.go [hunk 1]
```diff
@@ -2,7 +2,7 @@ package main
 
 import (
 	"flag"
-	hnd "myip/handler"
+	hnd "github.com/leesutcliffe/myip/handler"
```
Comment (leesutcliffe) [inline main.go]: Not required, see suggested changes

File: start.go [hunk 2]
```diff
@@ -3,7 +3,7 @@ package main
 import (
 	"errors"
 	"fmt"
-	hnd "myip/handler"
+	hnd "github.com/leesutcliffe/myip/handler"
```
Comment (leesutcliffe) [inline start.go]: Not required, see suggested changes

File: main.go (hunk 1 above)
Comment (andy-mcgrath) [reply main.go]: reverted and added a `go.mod` into the root of the project

**Review by leesutcliffe — APPROVED** [review]

**Pull request merged**